inst/rmarkdown/templates/trundler_template/skeleton/skeleton.Rmd

```diff
@@ -92,34 +135,43 @@ ggplot(bovril_250g_prices, aes(x = date)) +
 
 ```
 
-## Get the average prices for a range of products
+<br>
 
-To get the average prices for a particular product or product category, a little more manipulation is required.
+## Track average prices over time
 
-Let's say that you want to create an average price plot for single pineappples.
-
-First, you'll need to get the data for the products you want to aggregate.
+It can be useful to track how the average price of a product changes over time.    
```

Comment:
I was wondering, would it maybe make more sense to keep the same product here (or expand the net to include other Bovril products) for consistency? If we're thinking about a "report" then a report on Bovril prices makes sense but Bovril and Queen Pineapples... that's a stretch! :)